samples/apps/copilot-chat-app/webapp/src/styles.tsx

```diff
@@ -1,7 +1,7 @@
 import { GriffelStyle, tokens } from '@fluentui/react-components';
 
 export const CopilotChatTokens = {
-    backgroundColor: '#9c2153',
+    backgroundColor: '#000',
```

Comment:
undo? (unless there's a feature-related reason for this change)

Reply:
Nope, part of the other accidental commits.   Undone.